test/val/val_id_test.cpp

```diff
@@ -2103,7 +2103,271 @@ OpFunctionEnd
                         "base <id> (OpTypeFloat)."));
 }
 
-// TODO: OpInBoundsAccessChain
+// Valid: Access a float in a matrix using OpInBoundsAccessChain
```

Comment:
There are two schools of thought.
1. Tests should be dead simple straight line code.  This is the purist party line in Google.  It can lead to lots of duplication.  But within reason that duplication is worth price in test stimplicity.
2. Parameterize if you can.    This has the cost of extra complexity in the generic parts of the tests.

We've adopted the parameterize strategy in many of the tests in SPIRV-Tools.   There are reasonable arguments both ways.   This wholesale duplication is acceptable, but we're going to get another 2-fold increase in the next while for the Ptr* variants of access chains.  It's a bit ugly.

Probably not worth the refactor now.  But for next set of tests (not the AccessChain ones), let's try to use test parameterization.